### PR TITLE
use external NCEPLIBS-ip develop branch

### PR DESCRIPTION
**Pull request by webisu**
yet another version of nceplib/ip fix

Part of #115

### CODE REVIEW

**Comment by edwardhartnett** [comment]
Well we only need one version. ;-)

Shall we close the other PR?

**Comment by edwardhartnett** [comment]
I see that this is failing CI because the CI does not build ip. I will add that to this branch...

File: CMakeLists.txt [hunk 1]
```diff
@@ -93,18 +93,11 @@ endif()
 
 # If user wants to use NCEPLIBS-ip, find it and the sp library.
 message(STATUS "Checking if the user want to use NCEPLIBS-ip...")
-if(USE_IPOLATES EQUAL 1)
+if(USE_IPOLATES)
   find_package(ip CONFIG REQUIRED)
-  find_package(sp CONFIG REQUIRED)
-elseif(USE_IPOLATES EQUAL 3)
-  find_package(ip2 CONFIG REQUIRED)
+  list(APPEND definitions_list -DIPOLATES_LIB="ipolates_lib_d")
```
Comment (edwardhartnett) [inline CMakeLists.txt]: Why are you using the _d version of the library?

**Comment by webisu** [comment]
They are all the same with different versions of develop..  Close them all.

**Comment by edwardhartnett** [comment]
@webisu ok, let's keep this PR open and work on this branch until we get this working...

File: CMakeLists.txt (hunk 1 above)
Comment (edwardhartnett) [reply CMakeLists.txt]: Not intentional...

**Comment by AlysonStahl-NOAA** [comment]
@edwardhartnett @webisu Looks like things are working now. Did we want to move to using only two options for USE_IPOLATES (ON/OFF) in this pull request?

**Comment by webisu** [comment]
Yes,      The only reason there were two libraries was the need to support
rotated lat-lon grids using NCEP and WMO grid
templates.  A future nceplib/ip will support both templates.
Wesley

**Review by edwardhartnett — APPROVED** [review]

**Comment by edwardhartnett** [comment]
@AlysonStahl-NOAA you will be doing the merges in this repo.

Great job on this @webisu and @AlysonStahl-NOAA . Does this mean we can remove the internal copy of the ip code?

**Comment by AlysonStahl-NOAA** [comment]
> @AlysonStahl-NOAA you will be doing the merges in this repo.
> 
> Great job on this @webisu and @AlysonStahl-NOAA . Does this mean we can remove the internal copy of the ip code?

I don't believe the internal copy is referenced anywhere other than the makefile in the lib folder.

**Pull request merged**